src/main/frontend/src/api/SchemataRepository.js

```diff
@@ -0,0 +1,53 @@
+import Repository from '@/api/Repository'
+
+
+const resources = {
+    organizations: () => 'organizations',
```

Comment:
@wwerner Should we have a home/root with all the the nav links as hypermedia, or is this hardcoding ok?

Reply:
I think that would be nice and remove some coupling between client and server. But since currently both client and server live in the same project and the client application is served from the same backend, I wouldn't see it as urgent as of now.